hack/run-test.sh

```diff
@@ -73,7 +73,7 @@ main() {
     which ginkgo
     if [ $? -ne 0 ]; then
         echo "Downloading ginkgo tool"
-        go install -mod=mod github.com/onsi/ginkgo/v2/ginkgo@v2.6.1
+        go install -mod=mod github.com/onsi/ginkgo/v2/ginkgo@v2.17.1
```

Comment:
This change is not needed anymore:
https://github.com/openshift/cluster-node-tuning-operator/pull/1092